build/build.c

```diff
@@ -276,7 +279,11 @@ static int doBuildRequires(rpmSpec spec, int test)
 
  exit:
     freeStringBuf(sb_stdout);
-    free(output);
+    if (output) {
```

Comment:
Good catch, but this should simply be `argvFree(output)` instead.

Reply:
Thank you for the tip, I've made the relevant changes.